codalab/lib/bundle_cli.py

```diff
@@ -1362,6 +1364,15 @@ def do_upload_command(self, args):
 
             # Canonicalize paths (e.g., removing trailing /)
             sources = [path_util.normalize(path) for path in args.path]
+            # if sources is an array check the cumulative size with the users disk quota
```

Comment:
Not sure if "if sources is an array" is relevant? let's clarify this comment by saying something like "calculate size of sources"

Reply:
done